catalogue/webapp/components/WorkRedesign/WorkRedesign.js

```diff
@@ -13,33 +13,38 @@ import Layout10 from '@weco/common/views/components/Layout10/Layout10';
 import Layout12 from '@weco/common/views/components/Layout12/Layout12';
 import SearchBox from '@weco/common/views/components/SearchBox/SearchBoxV2';
 import SpacingSection from '@weco/common/views/components/SpacingSection/SpacingSection';
+import type {LicenseData} from '@weco/common/utils/get-license-info';
```

Comment:
I've started putting type inputs right at the top, follwoing:
* external modules types
* internal module types
* external modules
* internal modules

The only reason is so that VSC can auto import modules properly as it battles with `import type` being the last type.

Reply:
makes sense, yeah